examples/cdk/package.json

```diff
@@ -11,6 +11,7 @@
     "package": "echo 'Not applicable'",
     "test:unit": "npm run build && jest",
     "test:e2e": "echo 'To be implemented ...'",
+    "version": "npm install @aws-lambda-powertools/commons@0.4.0 @aws-lambda-powertools/logger@0.4.0 @aws-lambda-powertools/tracer@0.4.0 @aws-lambda-powertools/metrics@0.4.0",
```

Comment:
Why are we setting a specific version? And who is going to call `npm run version`?

Reply:
Examples are written against a certain version of our package. By definition its a version that has already been pushed to NPM.

Reply:
lerna version is calling version